tools/archives/pyvo_integration/astronomical_archives.xml

```diff
@@ -0,0 +1,164 @@
+<tool id="astronomical_archives" name="Astronomical archives" version="1.0.0">
+    <description>queries astronomical archives</description>
+    <requirements>
+        <requirement type="package" version="5.2.2">astropy</requirement>
+        <requirement type="package" version="1.4.1">pyvo</requirement>
+    </requirements>
+    <edam_operations>
+        <edam_operation>operation_0224</edam_operation>
+    </edam_operations>
+    <command>
+      <![CDATA[
+
+        mkdir fits;
+
+        #if $query_section.query_selection.query_type == 'none':
+
+          #if $archiveselection.archive_type == 'registry' :
+
+            python '$__tool_directory__/archivespyvo.py' '$output' '$file_type' $number_of_files '$archiveselection.archive_type' '$archiveselection.keyword' '$archiveselection.wavebands' '$archiveselection.service_type' '$query_section.query_selection.query_type'
```

Comment:
where does `archivespyvo.py` come from? It's not in the PR. Maybe it's an old name of the script?

Reply:
oh yea sorry forgot to change it, it's the name I used in my install to not conflict with the other ones I made.
I will push the change with the correct name and you can just rename it archive.py in the meantime if you want to try it

Reply:
I thought so, but it had to be fixed anyway, thanks.